app/build.gradle

```diff
@@ -75,4 +78,12 @@ dependencies {
 
     // TextDrawable
     implementation 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
+
+    //Kotlin dependencies
+    implementation "androidx.core:core-ktx:1.1.0"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.1.0"
```

Comment:
we are not using Kotlin extensions yet. so remove both lines

Reply:
Yeah, these were automatically added with Kotlin configuration. Will remove these.